docs/source/dev-docs.rst

```diff
@@ -154,6 +154,12 @@ the "stage" directory is used as DCSTOOL_HOME and DCSTOOL_USERDIR is the same de
     # to run with the java remote debugger enabled
     ant run -DdebugPort=5006
 
+    # to run with Java Flight Recorder
+    ant run -Dopendcs.jfr=true
```

Comment:
```suggestion
    ant run -Dopendcs.jfr=true   -Dopendcs.profile=/project1/my-nice.profile
```

Is this how the profile feature would be used?

Reply:
Yep.... apparently I not previously document that with the run target documentation.

Reply:
Made a minor update and typo fix.